benchmark/opperf/rules/default_params.py

```diff
@@ -431,7 +433,7 @@
                    "delta" : DEFAULT_DELTA,
                    "lr" : DEFAULT_LR,
                    "lrs" : DEFAULT_LRS,
-                   "wds" : DEFAULT_LRS,
+                   "wd" : DEFAULT_LR,
```

Comment:
Isn't wd represent for weight_decay? Why does it take the DEFAULT_LR here?

Reply:
using the same value as lr [0.1,0.5,0.9] 
so instead of creating additional variables I tried to reuse existing one as much as possible. can create a new var if needed?

Reply:
Yes, I think it's better to create a new var even if its value is the same as other var. It improves readability.

Reply:
Alright addressed